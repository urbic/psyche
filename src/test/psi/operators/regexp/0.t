@@ ?
@(.)\1@ ?
@\@@ ?
@a\\b@ ?
@\0\a\t\cI\n\f\r\\\u2026\x{2026}@ ?
