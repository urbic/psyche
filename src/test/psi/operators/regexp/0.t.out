~~
~(.)\1~
~\~~
~a\b~
~\0\a\t\t\n\f\r\……~
